sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -672,9 +672,6 @@ RTDeviceBinaryImage &ProgramManager::getDeviceImage(OSModuleHandle M,
   return *Img;
 }
 
-// TODO: getDeviceLibPrograms should also support Windows but
-// current implementation doesn't work on Windows when multiple
-// device libraries exist and this problem should be fixed.
```

Comment:
Could you please remove the unrelated changes?

Reply:
Several months ago, we found device library infrastructure can't work on Windows for multiple device libraries. We thought it was sycl runtime's bug so added the "TODO" here but it turned out the root cause was in underlying runtime and they have fixed this issue a bout 1 month ago. So we can directly remove the comments here without any code change.
Thank you very much.

Reply:
Already removed the unrelated change.